src/helpers/dataLoading/plans.ts

```diff
@@ -84,3 +95,13 @@ export async function loadOpenSRPPlan(
       displayError(err);
     });
 }
+
+/** Added to superset requests to specify only FI and Dynamic-Fi plans are required. */
+export const supersetFIPlansParamFilters = [
```

Comment:
Will this work like we need it to?  i.e. will it filter for plans that are either FI **or** DynamicFI?  Or will it attempt to filter for plans that are FI **and** DynamicFi?

In any case, I vote that we switch to using an **IN** filter.  SQL **IN** is usually faster for these kinds of filters i.e. where you are checking if a variable is in a number of values.  An added benefit is that it would potentially make this code much simpler by checking if the intervention type is in enabled plan types without the IFs.

Please see this comment for the available filters: https://github.com/onaio/canopy/issues/228#issuecomment-506699679

Reply:
> An added benefit is that it would potentially make this code much simpler by checking if the intervention type is in enabled plan types without the IFs

@moshthepitt 
does this mean we can have sth like `in [FI,IRS,MDA,MDA-Point,Dynamic-FI,Dynamic-IRS,Dynamic-MDA]`

Reply:
yeah the equivalent SQL will be something like `select ... from {the table} where intervention_type in (FI, DynamicFI)`;

Reply:
`57c3068 `

Reply:
https://github.com/onaio/reveal-frontend/pull/1277/commits/57c30684e6bfef33ecba4dc4ac1b40dfe7f9f828